api/views/yangSearch/yangSearch.py

```diff
@@ -445,6 +445,28 @@ def module_details(module: str, revision: t.Optional[str] = None, warnings: bool
     return response
 
 
+@bp.route('/draft-code-snippets/<draft_name>', methods=['GET'])
+def get_draft_code_snippets(draft_name: str) -> t.Optional[list[str, ...]]:
```

Comment:
`list[str, ...]` -> `list[str]`
Also please add a documentation comment.

Reply:
I realized `jsonify()` actually returns a string, so it should probably just be `t.Optional[str]`.

Reply:
I found out that ```jsonify()``` actually returns a ```Reponse``` object, fixed

Reply:
Ah, ok. Do add a docstring though, ideally all the endpoints where the response format isn't immediately obvious from the name should have one.

Reply:
Sorry, forgot about documentation, added now